<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FindWindowClass</class>
 <widget class="QMainWindow" name="FindWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1022</width>
    <height>705</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>查找</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="CheckBoxs">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>限定范围：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="CheckBoxName">
        <property name="text">
         <string>书名</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="CheckBoxAuthor">
        <property name="text">
         <string>作者</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="CheckBoxPublisher">
        <property name="text">
         <string>出版社</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Minimum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>550</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>查找内容：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="LineEditContent">
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="maxLength">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QRadioButton" name="RadioEqual">
          <property name="text">
           <string>等于</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">buttonGroup_3</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="RadioInclude">
          <property name="text">
           <string>包含</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">buttonGroup_3</string>
          </attribute>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Minimum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>600</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="SortControl">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="minimumSize">
         <size>
          <width>130</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>选择排序关键字：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="RadioName">
        <property name="text">
         <string>书名</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup_2</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="RadioDateAdded">
        <property name="minimumSize">
         <size>
          <width>90</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>进货日期</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup_2</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="RadioQty">
        <property name="text">
         <string>库存量</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup_2</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="RadioRetail">
        <property name="text">
         <string>零售价</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup_2</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="RadioWholesale">
        <property name="text">
         <string>批发价</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup_2</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>排序方式：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="RadioPositive">
        <property name="text">
         <string>A→Z</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="RadioBackward">
        <property name="text">
         <string>Z→A</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ButtonFind">
        <property name="text">
         <string>查找</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <widget class="QTableView" name="Table">
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1022</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup_3"/>
  <buttongroup name="buttonGroup_2"/>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
